pack/tcu/tsn.json

```diff
@@ -15,6 +50,58 @@
 		"type_code": "event",
 		"xp": 0
 	},
+	{
+		"code": "05112",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "mystic",
+		"flavor": "The body is but an extension of the mind.",
+		"illustrator": "John Pacer",
+		"name": "Sign Magick",
+		"pack_code": "tsn",
+		"position": 112,
+		"quantity": 2,
+		"skill_willpower": 1,
+		"slot": "Hand",
+		"text": "Fast.\nYou have 1 additional arcane slot, which can only be used to hold a <b>Spell</b> or <b>Ritual</b> asset.",
```

Comment:
I think this should be `[[Spell]]` or `[[Ritual]]` instead of the `<b>` tag since its a trait